services/hh/service.go

```diff
@@ -119,17 +119,26 @@ func (s *Service) WriteShard(shardID, ownerID uint64, points []models.Point) err
 
 func (s *Service) retryWrites() {
 	defer s.wg.Done()
-	ticker := time.NewTicker(time.Duration(s.cfg.RetryInterval))
-	defer ticker.Stop()
+	currInterval := time.Duration(s.cfg.RetryInterval)
```

Comment:
Should probably clamp the initial retry interval down to the Max as well here in case someone happened to misconfigure it.  e.g. they set a `retry-interval = 5m` but `retry-max-interval = 1m`.  The first retry would wait `5m` but subsequent ones would wait `1m`.